run/hello-broken/main.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2019 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask
+import json
+import os
+import sys
+
+app = Flask(__name__)
+
+# [START run_broken_service]
+@app.route("/", methods=["GET"])
+def index():
+    print("hello: received request.")
```

Comment:
Is there more useful logging we can put here and in line 44 to differentiate the two?

Reply:
Wanted to keep it the same as the other samples: [node.js](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/run/hello-broken/index.js#L20)